src/Orchard.Cms.Web/Modules/Orchard.Users/Views/Admin/Create.cshtml

```diff
@@ -7,12 +7,12 @@
 <form asp-action="Create" method="post">
     <div class="form-group" asp-validation-class-for="UserName">
         <label asp-for="UserName">@T["User name"]</label>
-        <span asp-validation-for="UserName" class="text-danger">@T["The user name is required."]</span>
+        <span asp-validation-for="UserName" class="text-danger"></span>
```

Comment:
I removed this error text because there are other error messages that can be returned for this field like `T[""User name '{0}' is already used.", user.UserName]` or `T["User name '{0}' is invalid, can only contain letters or digits.", user.UserName]`.
Problem is I don't know how to translate the message related with `[Required]` annotation in viewmodel.